src/components/example/EAccordion.tsx

```
@@ -0,0 +1,34 @@
+import {
+    Accordion,
```

ESLint

[error] Delete `··`

(prettier/prettier)